src/Aspire.Hosting.Azure.ServiceBus/AzureServiceBusExtensions.cs

```diff
@@ -96,13 +100,15 @@ public static IResourceBuilder<AzureServiceBusResource> AddAzureServiceBus(this
     /// <param name="builder">The Azure Service Bus resource builder.</param>
     /// <param name="name">The name of the topic.</param>
     /// <param name="subscriptions">The name of the subscriptions.</param>
-    public static IResourceBuilder<AzureServiceBusResource> AddTopic(this IResourceBuilder<AzureServiceBusResource> builder, [ResourceName] string name, string[] subscriptions)
+    /// <param name="configure">An optional method that can be used for customizing the <see cref="ServiceBusTopic"/>.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<AzureServiceBusResource> AddTopic(this IResourceBuilder<AzureServiceBusResource> builder, [ResourceName] string name, string[] subscriptions, Action<ServiceBusTopic>? configure = null)
```

Comment:
Do we need to add the Action here, if we also have it in the With methods?

Reply:
Should we consider obsoleting (or even removing??) the Add methods?

Reply:
@joperezr was also asking, I am ok to make these obsolete then. Wasn't sure it would count as breaking.

Reply:
We would still document obsoletion in our "what's new". I think we could obsolete them now. And then in 10 we could do a mass removal of obsolete APIs.

Reply:
Done, and created an issue in the docs repos.